packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -1925,7 +1925,7 @@ describe('plugin-meetings', () => {
                 rtpPackets: 0,
                 ssci: 0,
                 transmittedBitrate: 133.33333333333334,
-                transmittedFrameRate: 0,
+                transmittedFrameRate: 2,
```

Comment:
Doesn't really matter since we rewrote this test for the StatsAnalyzer move to internal-media-core, but just wondering what's the purpose of this change?

Reply:
@brycetham This change needed due to the total frames sent in a minute, which equals 1, being multiplied by 100, divided by 60, and rounded, resulting in a value of 2. Therefore, we need to update this in order to ensure that the tests work correctly for the SDK.

Reply:
Ah, that's right. So this is the equivalent of 0.02 frames per second, which seems to be correct.